src/track/trackmetadatataglib.cpp

```diff
@@ -1691,6 +1916,15 @@ void importTrackMetadataFromMP4Tag(TrackMetadata* pTrackMetadata, const TagLib::
         pTrackMetadata->refTrackInfo().setGrouping(grouping);
     }
 
+    QString work;
```

Comment:
Do you have here an external reference link as well. This would be a nice addition.

Reply:
Added link on top of file to avoid repetition